detection-rules/suspicious_request_for_quote_or_purchase.yml

```diff
@@ -0,0 +1,69 @@
+name: "Request for Quote or Purchase (RFQ|RFP) from suspicious sender"
+description: |
+  RFQ/RFP scams involve fraudulent emails posing as legitimate requests for quotations or purchases, often sent by scammers impersonating reputable organizations.
+  These scams aim to deceive recipients into providing sensitive information or conducting unauthorized transactions, often leading to financial loss, or data leakage.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound 
+  and 1 of (
+      (
+          (
+              length(recipients.to) == 0 or 
+              all(recipients.to, .display_name == "Undisclosed recipients")
+          )
+          and length(recipients.cc) == 0
+          and length(recipients.bcc) == 0
+      ),
+      (
+          strings.istarts_with(subject.subject, "RE:")
+          and (length(headers.references) == 0
+          or not any(headers.hops, any(.fields, strings.ilike(.name, "In-Reply-To"))))
+      ),
+      (
+          (
+              sender.email.domain.root_domain in $free_email_providers
+              and sender.email.email not in $recipient_emails
+          )
+          or 
+          (
+              sender.email.domain.root_domain not in $free_email_providers
+              and sender.email.domain.domain not in $recipient_domains
+          )
+      )
```

Comment:
may cause FPs since it's 1 of

Reply:
I think you're right... Pulling unsolicited increases accuracy